docs/user-guides/notifications/catalog/triggers.md

```diff
@@ -0,0 +1,90 @@
+# Triggers Catalog
+
+This document provides a catalog of recommended [triggers] for common use cases
+in Config Sync. These triggers are defined to use [templates from the catalog](./templates.md),
+but other templates can be provided (in the `send` block) as well.
+
+The set of [built-in functions](./functions.md) can be invoked from triggers.
+
+## on-reconciler-created
+
+The `RootSync`/`RepoSync` object is created. Triggers once for each newly created
+`RootSync`/`RepoSync` object.
+
+```yaml
+  trigger.on-reconciler-created: |
+    - when: True
+      oncePer: sync.metadata.name
+      send: [on-reconciler-created]
```

Comment:
replace `on-reconciler-created` with `reconciler-created`. We should use the template name here.

Same for other triggers.